src/main/java/com/amazon/ion/impl/IonReaderContinuableCoreBinary.java

```diff
@@ -89,6 +108,25 @@ class IonReaderContinuableCoreBinary extends IonCursorBinary implements IonReade
     // The symbol IDs for the annotations on the current value.
     private final IntList annotationSids;
 
+    // The IonReader-like MacroEvaluator that this core reader delegates to when evaluating a macro invocation.
+    private MacroEvaluatorAsIonReader macroEvaluatorIonReader = null;
+
+    // The core MacroEvaluator that this core reader delegates to when evaluating a macro invocation.
+    private MacroEvaluator macroEvaluator = null;
+
+    // Reads encoding directives from the stream.
+    private final EncodingDirectiveReader encodingDirectiveReader = new EncodingDirectiveReader();
+
+    // The text representations of the symbol table that is currently in scope, indexed by symbol ID. If the element at
+    // a particular index is null, that symbol has unknown text.
+    protected String[] symbols;
+
+    // The maximum offset into the 'symbols' array that points to a valid local symbol.
+    protected int localSymbolMaxOffset = -1;
```

Comment:
Moved from application-level because now either an Ion 1.0-style symbol table or an Ion encoding directive may determine the active symbol table.